theNewPAMs/paranormalminds_app
# Account activation

## Status
**IN DEVELOPMENT**

## Migrations
YES

## Description
After a user create an account, and email must be sent to their email with activation link. They my st be directed to the homepage and nitified that they have recieved and email with a link. 

When the link is clicked, the user's *activate* attribute must be changed to true thus activated. Now the user can login to view their profile.

## Related PRs
None

branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()


## Todos
- [ x ] Tests


## Deploy Notes
This works only on development env. For this to work in production env, we need a credit card to verify our details with Heroku. This is because the addon that sends emails need this verification done.

## Steps to Test or Reproduce
TO test this in dev env, create a user account. when the user is created, you will be directed to the homepage. To see the email sent, check the server log on the terminal where you are running rails s.

```sh
git pull --prune
git checkout account-activation
bundle; script/server
```

1. 

## Impacted Areas in Application
List general components of the application that this PR will affect:

* Only user create action will be affected but this feature.
